google-cloud-video-transcoder-v1/.repo-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-video-transcoder-v1/latest",
+    "distribution_name": "google-cloud-video-transcoder-v1",
+    "language": "ruby",
+    "name_pretty": "Google Cloud Video Transcoder V1 API",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "google-cloud-video-transcoder-v1 is the official client library for the Google Cloud Video Transcoder V1 API. Note that google-cloud-video-transcoder-v1 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-video-transcoder instead. See the readme for more details.",
```

Comment:
Is it OK that this library doesn't have the following?

* `product_documentation`
* `ruby-cloud-env-prefix`
* `ruby-cloud-product-url`

Reply:
Oh, good catch. We're missing some generator inputs on this one.